bioio/tests/test_bio_image.py

```diff
@@ -74,3 +81,32 @@ def test_bioimage_submission_data_reader_type_alignment(
 ) -> None:
     with pytest.raises(expected_exception):
         BioImage(image, reader=reader_class)
+
+
+def test_dump_plugins() -> None:
+    package_name = "bioio-czi"
+
+    try:
+        # Install the plugin
+        subprocess.check_call([sys.executable, "-m", "pip", "install", package_name])
```

Comment:
I think this test would be safer if you just made a Mock Reader that didn't have an author in it.  I'm honestly not sure how easy that would be to do (to act like it was installed), but it might be handy for other unit tests in this repo as well.

Reply:
added dummy plugin in [2c49cd8](https://github.com/bioio-devs/bioio/pull/59/commits/2c49cd8b4684ed8c28ed97c00caa294a453e554d)